spirit_msgs/msg/ControlInput.msg

```diff
@@ -0,0 +1,8 @@
+# This is a message to hold a spirit control input
+#
+# The control input is defined as the joint positions, joint velocities, body position, body velocity, body orientation, body angular velocity
```

Comment:
Is this comment correct? What gets stored in control input seems to differ in the below message declaration.

Reply:
@Kongx231

Reply:
That is not correct. I think I accidentally kept the state estimation comment in there. I think it should be the cartesian force in the world frame for each leg

Reply:
You should just be able to make the change and push it

Reply:
Soon as this gets modified feel free to pull in to master @jcnorby

Reply:
Never mind, changed.